Sources/GRPCOTelTracingInterceptors/Documentation.docc/Documentation.md

```diff
@@ -0,0 +1,56 @@
+# ``GRPCOTelTracingInterceptors``
+
+This module contains client and server tracing interceptors adhering to OpenTelemetry's 
+recommendations on tracing.
+
+## Overview
+
+You can read more on this topic at [OpenTelemetry's documentation](https://opentelemetry.io/docs). 
+Some relevant pages listing which attributes and events you can expect on your spans include:
+- [RPC Spans](https://opentelemetry.io/docs/specs/semconv/rpc/rpc-spans)
+- [gRPC conventions](https://opentelemetry.io/docs/specs/semconv/rpc/grpc)
+
+You can set up a client interceptor like so during your bootstrapping phase:
+
+```swift
+// Create the client interceptor
+let interceptor = ClientOTelTracingInterceptor(
+  serverHostname: "someserver.com",
+  networkTransportMethod: "tcp"
+)
+
+// Add it as an interceptor when creating your client
+let client = GRPCClient(
+  transport: someTransport, 
```

Comment:
I think we should use nondescript names for the things we're not instantiating to not draw any attention to them (or magic them into existence with something like `makeTransport()`):

```suggestion
  transport: transport, 
```